sources/packages/api/src/services/coe-denied-reason/coe-denied-reason.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Inject } from "@nestjs/common";
+import { RecordDataModelService } from "../../database/data.model.service";
+import { Connection } from "typeorm";
+import { COEDeniedReason } from "../../database/entities/coe-denied-reason.model";
+
+@Injectable()
+export class COEDeniedReasonService extends RecordDataModelService<COEDeniedReason> {
+  constructor(@Inject("Connection") private readonly connection: Connection) {
```

Comment:
Please remove the "private readonly" as it is not needed.
I noticed this along with many classes but it is probably generating a warning because the connection is not used anywhere besides the constructor.